src/__tests__/Search.test.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import renderer from "react-test-renderer";
+import Search from "../components/Search";
+
+describe("Search", () => {
+  const validProps = {
+    setFetchData: jest.fn(),
+    setErrorMessage: jest.fn(),
+    setSlideOn: jest.fn(),
+  };
+
+  test("Renders as expected", () => {
+    const rendered = renderer.create(
+      <Search
+        setFetchData={validProps.setFetchData}
+        setErrorMessage={validProps.setErrorMessage}
+        setSlideOn={validProps.setSlideOn}
+      />
+    );
+
+    expect(rendered).toMatchSnapshot();
+  });
+
+  test("Assert search-input is present", () => {
+    render(
+      <Search
+        setFetchData={validProps.setFetchData}
+        setErrorMessage={validProps.setErrorMessage}
+        setSlideOn={validProps.setSlideOn}
+      />
+    );
+
+    expect(screen.getByRole("textbox")).toHaveClass("search-input");
```

Comment:
Could we search for something the user could see here instead, like some placeholder text?